tools/interop_matrix/run_interop_matrix_tests.py

```diff
@@ -197,17 +198,17 @@ def run_tests_for_lang(lang, runtime, images):
         'grpc_interop_matrix',
         suite_name,
         str(uuid.uuid4()))
+
+    if not args.keep:
```

Comment:
How much space do we have on the VM?  Instead of per runtime cleanup, I feel we should do per language.

Reply:
Mounted on Avail
/tmpfs       50G
/            18G

We are not sure how much exactly is available for us. What's the benefit of keeping them around after they're done with the tests?

Reply:
It is for caching, as there might be shared layers between current image and next one to download.

Reply:
if 50G is not enough, it is quite easy to reconfigure the size on kokoro btw (not saying we should do that, just saying it's easy). See cl/175488640 as a demo.